config/file_config.go

```diff
@@ -350,6 +350,12 @@ func (f *fileConfig) validateGeneralConfigs() error {
 	default:
 		return fmt.Errorf("invalid CacheOverrunStrategy: '%s'", st)
 	}
+
+	if st != "impact" {
+		if f.GetStressReliefConfig().Mode == "monitor" || f.GetStressReliefConfig().Mode == "always" {
+			return fmt.Errorf("invalid CacheOverrunStrategy for StressReliefMode: '%s'", st)
+		}
+	}
```

Comment:
```suggestion
	if st != "impact" && (f.GetStressReliefConfig().Mode == "monitor" || f.GetStressReliefConfig().Mode == "always"){
		return fmt.Errorf("invalid CacheOverrunStrategy for StressReliefMode: '%s'", st)
	}
```

Reply:
Yeah I wrote it this way at first and did not prefer it, but I will change it if you prefer the longer conditional.